codec_specs.md

```diff
@@ -182,6 +182,26 @@ Description: The Matroska video stream will contain a demuxed Elementary Stream
 
 Initialization: none
 
+### V_MPEG4/ISO/AVC
+
+Codec ID: V_MPEG4/ISO/AVC
+
+Codec Name: AVC/H.264
+
+Description: Individual pictures of AVC/H.264 stored as described in [@!ISO.14496.15].
```

Comment:
``` draft-ietf-cellar-codec-04.xml(0): Error: IDREF attribute target references an unknown ID "ISO.14496.15", at None
draft-ietf-cellar-codec-04.xml(0): Error: IDREF attribute target references an unknown ID "ISO.14496.15", at None
draft-ietf-cellar-codec-04.xml(0): Error: IDREF attribute target references an unknown ID "ISO.14496.15", at None
draft-ietf-cellar-codec-04.xml(0): Error: IDREF attribute target references an unknown ID "ISO.14496.15", at None
draft-ietf-cellar-codec-04.xml(0): Error: IDREF attribute target references an unknown ID "ISO.14496.15", at None
draft-ietf-cellar-codec-04.xml(0): Error: IDREF attribute target references an unknown ID "ISO.14496.15", at None
draft-ietf-cellar-codec-04.xml(3): Error: Invalid document before running preptool.
```

This results in [errors in the CI](https://github.com/cellar-wg/matroska-specification/pull/377/checks?check_run_id=699125042). I guess this reference is not possible. We'd have to create it ourselves, like this [file in EBML](https://raw.githubusercontent.com/cellar-wg/ebml-specification/master/rfc_backmatter.markdown).

Reply:
I need help here, I added "ISO.14496-15.2014" (BTW, I changed the ID in order to reference the date) to the backmatter but I still get the error.

Reply:
You put it in ` rfc_backmatter_matroska.md` instead of `rfc_backmatter_codec.md`.
And you used the `—` character which is UTF-8, forbidden in RFCs.